packages/webapp/src/containers/saga.js

```diff
@@ -596,10 +594,6 @@ export function* fetchAllSaga() {
     put(getCertificationSurveys()),
     put(getAllSupportedCertifications()),
     put(getAllSupportedCertifiers()),
-    put(getRevenueTypes()),
-    put(getFarmExpenseType()),
-    put(getSales()),
-    put(getExpense()),
```

Comment:
I figured if this isn't needed for anything else other than the Finances page, we are already fetching this data and waiting for it when the Finances page loads so no need to pre-fetch it here.